packages/contracts/test/framework/dao/dao-factory.ts

```diff
@@ -164,7 +170,7 @@ describe('DAOFactory: ', function () {
       // @ts-ignore
       hre,
       'DAOFactory',
-      ['DAORegistry', 'PluginSetupProcessor', 'DAO']
+      ['DAORegistry', 'PluginSetupProcessor', 'src/core/dao/DAO.sol:DAO']
```

Comment:
Is this `'src/core/dao/DAO.sol:DAO'` necessary?
Isn't there a better way than the `getMergedABI` approach?